app/ante/tx_size_test.go

```diff
@@ -0,0 +1,189 @@
+package ante_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/v3/app"
+	"github.com/celestiaorg/celestia-app/v3/app/ante"
+	"github.com/celestiaorg/celestia-app/v3/pkg/appconsts"
+	v3 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v3"
+	testutil "github.com/celestiaorg/celestia-app/v3/test/util"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/crypto/types/multisig"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	authante "github.com/cosmos/cosmos-sdk/x/auth/ante"
+	xauthsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/stretchr/testify/require"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+func setup() (*app.App, sdk.Context, client.Context, sdk.AnteHandler, error) {
+	app, _, _ := testutil.NewTestAppWithGenesisSet(app.DefaultConsensusParams())
```

Comment:
Q: If we're using `DefaultConsensusParams` that will mean we're using app version 3 right? Has this been updated yet?

Reply:
No, it hasn't been updated yet but I left it as DefaultConsensusParams because we're planning to do so.